go-controller/pkg/ovn/obj_retry.go

```diff
@@ -711,6 +797,103 @@ func (oc *Controller) updateResource(objectsToRetry *retryObjs, oldObj, newObj i
 			extraParameters.portGroupIngressDenyName,
 			extraParameters.portGroupEgressDenyName,
 			newObj)
+	case factory.EgressIPType:
+		oldEIP := oldObj.(*egressipv1.EgressIP)
+		newEIP := newObj.(*egressipv1.EgressIP)
+		if err := oc.reconcileEgressIP(oldEIP, newEIP); err != nil {
+			return fmt.Errorf("unable to update EgressIP: %s, err: %v", newEIP.Name, err)
+		}
+		return nil
+	case factory.EgressIPNamespaceType:
+		oldNamespace := oldObj.(*kapi.Namespace)
+		newNamespace := newObj.(*kapi.Namespace)
+		if err := oc.reconcileEgressIPNamespace(oldNamespace, newNamespace); err != nil {
+			return fmt.Errorf("unable to update egress IP matching namespace: %s, err: %v", newNamespace.Name, err)
+		}
+		return nil
+	case factory.EgressIPPodType:
+		oldPod := oldObj.(*kapi.Pod)
+		newPod := newObj.(*kapi.Pod)
+		if util.PodCompleted(newPod) {
+			if err := oc.reconcileEgressIPPod(oldPod, nil); err != nil {
+				return fmt.Errorf("unable to delete egress IP matching completed pod: %s/%s, err: %v",
+					oldPod.Name, oldPod.Namespace, err)
+			}
+		} else {
+			if err := oc.reconcileEgressIPPod(oldPod, newPod); err != nil {
+				return fmt.Errorf("unable to update egress IP matching pod: %s/%s, err: %v", newPod.Name, newPod.Namespace, err)
+			}
+		}
+		return nil
+	case factory.EgressNodeType:
+		oldNode := oldObj.(*kapi.Node)
+		newNode := newObj.(*kapi.Node)
+		// Initialize the allocator on every update,
+		// ovnkube-node/cloud-network-config-controller will make sure to
+		// annotate the node with the egressIPConfig, but that might have
+		// happened after we processed the ADD for that object, hence keep
+		// retrying for all UPDATEs.
+		if err := oc.initEgressIPAllocator(newNode); err != nil {
+			klog.V(5).Infof("Egress node initialization error: %v", err)
```

Comment:
@flavio-fernandes , @kyrtapz can we safely ignore this error (as we did in the original code) and go on with the rest of the update?

Reply:
this can happen if we have bad eIP config so retry will never recover or the node subnet annotation is missing which also retry can't help once condition is fixed EgressNode watcher will tick and update eIP cache for that node, now regarding fall through and continue probably wasted effort but in the sametime we don't want retry logic to kick here either

Reply:
but shouldnt this be at least a Warn if not an Error message?